{
  "id": "threatAA02_ConnectionLostToRemoteService",
  "metadata": {
    "ns:textReviewContext": "Cloud application uses a remote service",
    "ns:textName": "AA02 Connection Lost To Remote Service",
    "ns:textReviewSolution": "Ensure fault tolerance of the network connection; Replace the remote service by a local one;",
    "ns:textURL": "",
    "ns:textReviewProblem": "Connection lost to the remote service",
    "ns:textReference": "BNA-3: Loss of wide-area network connectivity... [SECCRIT]"
  },
  "organization": {
    "ns:usesPattern": [],
    "ns:hasGroup": [],
    "ns:hasType": ["ns:type_ThreatPattern"],
    "ns:hasTemplate": [],
    "ns:relatesTo": [],
    "ns:isChildOf": []
  },
  "common": {
    "ns:hasArchitecturalLayer": [],
    "ns:hasResponseType": [],
    "ns:hasDomain": [],
    "ns:hasSecurityLevel": [],
    "ns:hasLifecycleStage": [],
    "ns:hasConstraintLevel": []
  },
  "context": {
    "ns:hasAffectedFunction": [],
    "ns:hasAffectedComponent": ["su:comp_CloudApplication"]
  },
  "security": {
    "ns:hasThreat": ["ns:threat_txMessageFlooding", "ns:threat_txResourceExhaustion"],
    "ns:hasAggressor": ["su:comp_ExternalService"],
    "ns:hasAggressorRole": ["ns:role_Server"],
    "ns:hasWeakness": [],
    "ns:hasSTRIDE": ["ns:STRIDE_Denial_of_Service"],
    "ns:hasSecurityConcern": []
  }
}